cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/DefaultNamespaceAdmin.java

```diff
@@ -144,10 +152,6 @@ public synchronized void create(final NamespaceMeta metadata) throws Exception {
       authorizerInstantiator.get().grant(namespace, principal, ImmutableSet.of(Action.ALL));
     }
 
-    // store the meta first in the namespace store because namespacedlocationfactory need to look up location
-    // mapping from namespace config
-    nsStore.create(metadata);
```

Comment:
Because @bdmogal changed NamespacedLocationFactory where now it take the meta itself and does not do look up when we create a namespace in earlier PR. That was the reason why we moved it above and since its not needed anymore we are moving where it was.